perfkitbenchmarker/configs/dataflow_wordcount.yaml

```diff
@@ -0,0 +1,85 @@
+#################################
```

Comment:
Does this get used somewhere?

Reply:
This is the config YAML I use to run the benchmark e.g. `./pkb.py --cloud=GCP --project=$PROJECT_ID --benchmark_config_file=dataflow_wordcount.yaml` . I found it's a learning curve and tricky to get these flags right for new PKB users. Therefore providing this as a self-contained example which was referenced in the talk.

Reply:
Awesome! Thank you for the addition.